Feature/geolocation (#267)
infinum/eightshift-forms: src/Hooks/FiltersGeolocation.md

	return __DIR__ . \DIRECTORY_SEPARATOR . 'geoip.phar';
}
```

## Provide custom geolocation cookie name.

> This filter provides you with the ability to provide custom cookie name for geolocation.

Reviewer: ```suggestion
This filter enables providing custom cookie name for geolocation.
```

**Global variable alternative:**
`ES_GEOLOCATION_COOKIE_NAME`

**Filter name:**
`es_forms_geolocation_cookie_name`

**Filter example:**
```php
// Geolocation cookie name.
add_filter('es_forms_geolocation_cookie_name', [$this, 'getGeolocationCookieName']);

/**
 * Geolocation cookie name.
 *
 * @return string
 */
public function getGeolocationCookieName(): string
{
	return 'esForms-country';
}
```

## Provide custom WP-Rocket advanced-cache.php function.

> This filter provides you with the ability to provide custom function in WP-Rocket plugin activation process.

Reviewer: ```suggestion
This filter enables providing custom function in WP-Rocket plugin activation process.
```
